packages/esm-patient-registration-app/src/patient-registration/form-manager.ts

```diff
@@ -58,6 +58,7 @@ export default class FormManager {
       },
       {
         id: patientUuid,
+        // @ts-ignore
```

Comment:
This should probably not be a TS error. It is a fairly new API in esm-framework though. Could you check whether updating the packages solves the issue? (`rm -rf node_modules && rm yarn.lock && npx lerna bootstrap`)